tools/dropdown.js

```diff
@@ -1,40 +1,44 @@
-/* exported toggleList */
-/* When the user clicks on a button,
-toggle between hiding and showing the dropdown content
-if off close all the others before turning on */
-function closeAll() {
-    "use strict";
-    var dropdowns = document.getElementsByClassName("dropdown-content");
-    var i, openDropdown;
-    for (i = 0; i < dropdowns.length; i++) {
-        openDropdown = dropdowns[i];
-        if (openDropdown.classList.contains("dropdown-show")) {
-            openDropdown.classList.remove("dropdown-show");
-        }
-    }
-}
+"use strict";
+window.addEventListener("DOMContentLoaded", () => {
+    /*
+     * When the user clicks on a button,
+     * toggle between hiding and showing the dropdown content
+     * if off close all the others before turning on
+     */
 
-function toggleList(id) {
-    "use strict";
-    var thisDropdown = document.getElementById(id);
-    var turnOn = !thisDropdown.classList.contains("dropdown-show");
-    closeAll();
-    if (turnOn) {
-        thisDropdown.classList.add("dropdown-show");
+    function closeAll() {
+        var dropdowns = document.getElementsByClassName("dropdown-content");
+        for (let openDropdown of dropdowns) {
+            if (openDropdown.classList.contains("dropdown-show")) {
+                openDropdown.classList.remove("dropdown-show");
```

Comment:
It doesn't seem you need the if at all: https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove
```
 If the string is not in the list, no error is thrown, and nothing happens. 
```

Reply:
I've removed it.